src/middlewares/req-handler.js

```
@@ -0,0 +1,11 @@
+const reqHandler = (anyFunction) => {
+  return async (req, res, next) => {
+    try {
+      await anyFunction(req, res, next);
+    } catch (error) {
+      console.error({ Error: error });
```

This swallows the error here, it will not return the error to the client.

---

done